src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
```

Comment:
Good job! Now just add variables for all the magic numbers (40, 50, 20).

Reply:
Agree

Reply:
Also you can move do this calculation before all the checks to not repeat yourself in every return statement